test/fakeOrigin/shard.sh

```diff
@@ -0,0 +1,84 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+#!/bin/bash
+set -ex
+
+# brew install ffmpeg --with-rtmp-dump
+# brew install mp4box
+
+# Sample Invokation:
+#   ./shard.sh ./example/video/kelloggs.mp4 ./out/shard.hls.vod HLS_VOD
+#   ./shard.sh ./example/video/kelloggs.mp4 ./out/shard.hls.live HLS_LIVE
+#   ./shard.sh ./example/video/kelloggs.mp4 ./out/shard.dash.vod DASH_VOD
+#   ./shard.sh ./example/video/kelloggs.mp4 ./out/shard.dash.live DASH_LIVE
+
+
+
+src=$1
+destination=$2
+format=$3
+filename=$(basename "${destination}")
+
```

Comment:
you probably wanna bail then if `$#` is less than 3

Reply:
Addressed with new commits